zoom-recording-downloader.py

```diff
@@ -37,8 +37,8 @@
 
 # Start date now split into YEAR, MONTH, and DAY variables (Within 6 month range)
 RECORDING_START_YEAR = 2021
-RECORDING_START_MONTH = 1
-RECORDING_START_DAY = 1
+RECORDING_START_MONTH = 12
+RECORDING_START_DAY = 29
 RECORDING_END_DATE = date.today()
 # RECORDING_END_DATE = date(2021, 8, 1)
 DOWNLOAD_DIRECTORY = 'downloads'
```

Comment:
can you take directory path from command line as well?

Reply:
The downloads path is fixed because uploading on youtube will be from that file. If directory path is user input then might be a chance of error and also the same directory need to be mentioned in the main.py